### ISSUE TITLE
ListItem crashes unexpectedly on Web

### ISSUE THREAD

**Issue by deb-95** [react-native-elements/react-native-elements#2541]
I upgraded React Native Elements to v2.3.1 and expo-cli to 3.25.1.
I expected the web application to work with warnings about deprecated props on ListItem but when I start the app in web version an unexpected error occures in ListItem:
`
ListItem.js:41 Uncaught TypeError: Cannot read property 'grey' of undefined
    at renderIcon (ListItem.js:41)
    at ListItem (ListItem.js:220)
`

Here is an example on Snack to reproduce.
https://snack.expo.io/yzVlD7RBS

I don't know if it helps but debugging it turned out the theme.colors.platform (at line 41 of ListItem) object doesn't contain any "default" property
![image](https://user-images.githubusercontent.com/44870331/91822222-bc9d4000-ec37-11ea-9901-30512a0fac10.png)

These are the lines where I put the breakpoint in the ListItem file and where the error occurs:
`
const renderIcon = (content, theme) =>
  renderNode(Icon, content, {
    color: Platform.select(theme.colors.platform).grey,
    size: 24,
  });
`

**Comment by Stolson** [comment]
Also having this issue with 2.3.1, but with a different error:

`react_native_web_dist_exports_Platform__WEBPACK_IMPORTED_MODULE_10__.default.select(...) is undefined`

I've tried `react-native-web` from 0.11.7 - 0.13.0 with Expo 38 but haven't had much luck.

**Comment by alexkyse1** [comment]
I'm also facing this issue.

**Comment by mikehenken** [comment]
**Really easy fix!**

I resolved this locally by adding grey to the theme for platform "default". 

```
import { ListItem, ThemeProvider} from 'react-native-elements';
const theme = {  
  colors: {
    platform: {
      "default": {
        "grey": "#FFF"
      }
    }
  }
};

<ThemeProvider theme={theme}>
  <View>
    {
      list.map((item, i) => (
        <ListItem key={i} bottomDivider>
          <Icon name={item.icon} />
          <ListItem.Content>
            <ListItem.Title>{item.title}</ListItem.Title>
          </ListItem.Content>
          <ListItem.Chevron />
        </ListItem>
      ))
    }
  </View>
</ThemeProvider>
```

Apparently someone put grey0 instead of grey as platform default. IOS & Android use grey, default uses grey0. See:
https://github.com/react-native-elements/react-native-elements/blob/ed6e90d55f18f371ce606019c28f937f816ea3a8/src/list/ListItem.js#L41
And
https://github.com/react-native-elements/react-native-elements/blob/62066e442e49630bb8490f5eb9f24412024945a1/src/config/colors.js#L8

**Comment by mikehenken** [comment]
**Really easy fix!**

I resolved this locally by adding grey to the theme for platform "default". 

```javascript
import { ListItem, ThemeProvider} from 'react-native-elements';
const theme = {  
  colors: {
    platform: {
      "default": {
        "grey": "#FFF"
      }
    }
  }
};

<ThemeProvider theme={theme}>
  <ListItem bottomDivider>
    <ListItem.Content>
      <ListItem.Title>
        Tracks
      </ListItem.Title>
    </ListItem.Content>
    <ListItem.Chevron />
  </ListItem>
</ThemeProvider>
```

Apparently someone put grey0 instead of grey as platform default. IOS & Android use grey, default uses grey0. See:
https://github.com/react-native-elements/react-native-elements/blob/ed6e90d55f18f371ce606019c28f937f816ea3a8/src/list/ListItem.js#L41
And
https://github.com/react-native-elements/react-native-elements/blob/62066e442e49630bb8490f5eb9f24412024945a1/src/config/colors.js#L8

**Comment by danielnathanael** [comment]
> 
> 
> **Really easy fix!**
> 
> I resolved this locally by adding grey to the theme for platform "default".
> 
> ```js
> import { ListItem, ThemeProvider} from 'react-native-elements';
> const theme = {  
>   colors: {
>     platform: {
>       "default": {
>         "grey": "#FFF"
>       }
>     }
>   }
> };
> 
> <ThemeProvider theme={theme}>
>   <ListItem bottomDivider>
>     <ListItem.Content>
>       <ListItem.Title>
>         Tracks
>       </ListItem.Title>
>     </ListItem.Content>
>     <ListItem.Chevron />
>   </ListItem>
> </ThemeProvider>
> ```
> 
> Apparently someone put grey0 instead of grey as platform default. IOS & Android use grey, default uses grey0. See:
> 
> https://github.com/react-native-elements/react-native-elements/blob/ed6e90d55f18f371ce606019c28f937f816ea3a8/src/list/ListItem.js#L41
> 
> And
> 
> https://github.com/react-native-elements/react-native-elements/blob/62066e442e49630bb8490f5eb9f24412024945a1/src/config/colors.js#L8

Thank you it works, I'll use this solution while waiting for bug to be fixed.

**Comment by imakshar** [comment]
@flyingcircle I make PR #2561, this is working fine on the web too after this.
I hope it's one of the solutions to milestone 3.0.0 :)

**Comment by pranshuchittora** [comment]
Closing the issue. Fixed in #2561 
If the issue still persists then feel free to reopen the issue.
Happy Hacking :)
